content/kots-cli/download.md

```diff
@@ -5,4 +5,24 @@ title: kots download
 weight: 90120
 ---
 
-Once an application is running and has the Admin Console (kotsadm) deployed with it, you can run `kots download` to retrieve a copy of the application manifests from the cluster, and store them in a specific directory structure on your workstation.
+Retrieves a copy of the application manifests from the cluster, and store them in a specific directory structure on your workstation. Requires a running KOTS application with the Admin Console (kotsadm).
+
+### Usage
+```bash
+kubectl kots download [app-slug] [flags]
+```
+* _Replace `[app-slug]` with the "slug" of your KOTS application_ 
+* _Provide `[flags]` according to the table below_ 
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `--dest` | string |        the directory to store the application in (default "/Users/austin") |
+| `-h, --help`   |  |           help for download |
+| `--kubeconfig` | string |  the kubeconfig to use (default "/Users/austin/.kube/config") |
+| `-n, --namespace` | string |    the namespace to download from (default "default") |
+| `--overwrite` |   |       overwrite any local files, if present |
```

Comment:
this is a `bool` type.

Reply:
A quick glance, I think all of the missing types in this doc are bool types

Reply:
@marccampbell For all the issues with KOTS command-line flags, these were copied from kots --help, so if we want to change it here, we should probably also change it there. 

As for the missing types, I was thinking that when unspecified, it is not followed by a value. For example, I think usage is '--overwrite' and not '--overwrite true'

Reply:
I'm inclined to keep this as-is, and update if we update the "--help" command.